tests/platform_tests/api/test_module.py

```diff
@@ -120,6 +120,9 @@ def test_get_model(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost,
             if self.skip_absent_module(i,platform_api_conn):
                 continue
             model = module.get_model(platform_api_conn, i)
+            name = module.get_name(platform_api_conn, i)
```

Comment:
isn't this already done by line 120 already?
 if self.skip_absent_module(i,platform_api_conn):
                continue
is performing exactly what you are trying to add ?  So why that did not skip over the absent modules?
Can you check your change again?